src/pykoala/data_container.py

```diff
@@ -810,10 +810,11 @@ def science_fibres(self):
     def __init__(self, **kwargs):
         assert ('wavelength' in kwargs)
         assert ('intensity' in kwargs)
+        assert ('fibre_diameter' in kwargs)
```

Comment:
Maybe we need to rethink the way we check the arguments. I do not like the idea of having these many asserts...